tests/test_main.py

```diff
@@ -82,6 +82,14 @@ def test_atbash():
         "Svool nb mznv rh yvv zmw R orpv wlt zmw zkkov zmw givv",
     )
     assert res.lower() == answer_str.lower()
+  
+
+def test_tap_code():
+    res = decrypt(
+        Config().library_default().complete_config(),
+        "5,2 3,4 4,2 1,3 4,3",
```

Comment:
This string should be the tap-code-encrypted version of the `answer_str` specified in the header of the tests file: `Hello my name is bee and I like dog and apple and tree`

Reply:
For the test to pass, you need a solution that correctly substitutes "1,3" for K (since k occurs in the test sentence).

Reply:
Unrelated idea: Switch out the current `answer_str` for a pangram such as `The quick brown fox jumps over the lazy dog` or `Sphinx of black quartz, judge my vow`. This would help detect errors in output that only affect certain characters - which, for example, my galactic decoder is struggling with.

Reply:
See here: https://en.wikipedia.org/wiki/The_quick_brown_fox_jumps_over_the_lazy_dog#Computer_usage